src/protocol.rs

```diff
@@ -287,7 +287,18 @@ impl ParticipantConfig {
     }
 }
 
-/// An identifier corresponding to a [`Participant`].
+/// An identifier for a [`Participant`].
+///
+/// All [`Participant`]s in a session must agree on the
+/// [`ParticipantIdentifier`]s. That is, these are not local identifiers for a
+/// single [`Participant`]; they are global identifiers for the [`Participant`]s
```

Comment:
I'm being kind of weird about this because `ParticipantIdentifier`s are unique per participant, but not necessarily per session (e.g. you can keep the same pid across keygen, auxinfo, and several presigns). I'll just state that clearly, though.